__tests__/app.test.js

```diff
@@ -32,6 +32,51 @@ describe('/api/topics', () => {
   });
 });
 
+describe('/api/articles', () => {
+  it('200: should return the correct article object corresponding to the provided id', () => {
+    return request(app)
+      .get('/api/articles/5')
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toMatchObject({
+          article_id: 5,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
```

Comment:
You can be more specific here, if you've gone to articles/5 then you could add more specific properties. 

I would say however that the most important to match is article_id and you've done that. So I'll leave it upto you!